stock_inventory_line_price/models/stock_inventory.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Esther Martín - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+import openerp.addons.decimal_precision as dp
+
+
+class StockInventoryLine(models.Model):
+    _inherit = 'stock.inventory.line'
+
+    standard_price = fields.Float('Cost Price',
+                                  digits=dp.get_precision('Product Price'))
+
+    @api.model
+    def _resolve_inventory_line(self, inventory_line):
+        inventory_line.product_id.standard_price = \
```

Comment:
I'm not completely sure, but if the concept is "there are 10 units of ProductA with a cost of 0'43", in that case it must not compute the average cost, it must rewrite it. @anajuaristi can you give as a point?